lib/rules/prefer-object-has-own.js

```diff
@@ -87,6 +87,13 @@ module.exports = {
                                 return null;
                             }
 
+                            const tokenJustBeforeNode = sourceCode.getTokensBefore(node, 1)[0];
+
+                            // for https://github.com/eslint/eslint/pull/15346#issuecomment-991417335
+                            if (tokenJustBeforeNode && tokenJustBeforeNode.type === "Keyword" && !sourceCode.isSpaceBetween(node, tokenJustBeforeNode)) {
```

Comment:
This is not enough, see https://github.com/sindresorhus/eslint-plugin-unicorn/blob/32c20196528c28d9dd6eb1f3beb15f5528480418/rules/fix/fix-space-around-keywords.js#L4

Reply:
Thanks.